services/keyboard/src/org/chromium/mojo/keyboard/KeyboardServiceImpl.java

```diff
@@ -17,20 +17,12 @@
  * Android implementation of Keyboard.
  */
 public class KeyboardServiceImpl implements KeyboardService {
```

Comment:
Should we move this into `sky/services` now that it is diverging from the modroid implementation?